sourcecode/apis/contentauthor/Dockerfile

```diff
@@ -74,26 +76,41 @@ RUN set -eux; \
 
 
 FROM buildresult AS cron
+ENV LOG_CHANNEL=stderr \
```

Comment:
Unfortunately, this would prevent the values from being changed via .env. We really need separate base stages for dev and prod containers (should be done in a separate issue IMO.)

Reply:
`APP_ENV=production` and `APP_DEBUG=false` are already defaults so it shouldn't be necessary to set it again here. `LOG_CHANNEL=stderr` should really be default for all of our envs. I don't understand what that would prevent us doing?